### PR TITLE
executeWorkbench args

### PR DESCRIPTION
**Pull request by meguia-sfdc**
Currently, `VSCodeCommands` interface doesn't allow to pass parameters to `executeWorkbench` method. We add the parameters args to its signature.

~~Additionally, mocha invert flag is set to its default value.~~

### CODE REVIEW

File: test/wdio.conf.ts [hunk 1]
```diff
@@ -192,8 +192,7 @@ export const config: Options.Testrunner = {
     mochaOpts: {
         ui: 'bdd',
         timeout: 60000,
-        grep: grep.join('|'),
-        invert: true
```
Comment (christian-bromann) [inline test/wdio.conf.ts:196]: How is this related to the original issue?

**Review by christian-bromann — COMMENTED** [review]
Good catch, one comment

File: test/wdio.conf.ts (hunk 1 above)
Comment (meguia-sfdc) [reply test/wdio.conf.ts:196]: Not related at all. I did this to run tests in all platforms already set up in Actions. I was going to add a comment to check if there is something wrong about running tests in GitHub actions. Are we fine with it?

Comment (christian-bromann) [reply test/wdio.conf.ts:196]: it is currently causing the test suite to fail because some web tests should not run in CI due to issues with the environment.

Comment (meguia-sfdc) [reply test/wdio.conf.ts:196]: Reverting.

**Review by christian-bromann — APPROVED** [review]
👍

File: test/specs/basic.e2e.ts [hunk 2]
```diff
@@ -116,6 +116,22 @@ describe('WDIO VSCode Service', () => {
                 timeoutMsg: 'Could not find another custom notification'
             })
         })
+
+        it('can send parameters to VSCode API invocation', async () => {
```
Comment (christian-bromann) [inline test/specs/basic.e2e.ts:120]:
```suggestion
        it('can send parameters to VSCode API invocation @skipWeb', async () => {
```

**Review by christian-bromann — COMMENTED** [review]
We need to skip the test for web because that is not yet supported, see #24

File: test/specs/basic.e2e.ts (hunk 2 above)
Comment (meguia-sfdc) [reply test/specs/basic.e2e.ts]: You mean #23 , right?
